modules/ROOT/pages/connected-apps-devs-end-users.adoc

```diff
@@ -0,0 +1,20 @@
+= Connected Apps for Developers and End Users
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Developers
+
+Third-party developers who have their own websites or web apps can allow users to log in using the *Log in with Anypoint Platform* option. For seamless implementation, there is a standard button that can be integrated into the front end of your application with a pre-built template and set of components. For more information about how to embed the button, refer to the  https://github.com/anypoint-web-components/anypoint-signin/tree/master[developer tutorial] and the http://anypoint-signin-test.surge.sh/[developer playground]. 
+
+In addition to enabling new flows to authenticate users, developers can enable or disable clients and view client usage.
+For example, developers can see how many organizations are using the app and how many users have access.
+
+== End Users
+
+End users can log into a third-party applications using their Anypoint credentials.
+They can can authorize access to their account when the following message appears:
+
+// screenshot of authorize user page https://qax.anypoint.mulesoft.com/login/authorize?state=eyJjbGllbnQiOnsiY2xpZW50X2lkIjoiZWEzYzJiODFkMDEyNGI0Mjk1NDYyM2RhMWM0MzQ5ZWIifSwidXNlciI6eyJ1c2VybmFtZSI6Im9yZ2FuaXphdGlvbl9vd25lciJ9LCJvcmlnaW4iOiIvYXBpL3YyL29hdXRoMi9hdXRob3JpemU%2FY2xpZW50X2lkPWVhM2MyYjgxZDAxMjRiNDI5NTQ2MjNkYTFjNDM0OWViJnJlZGlyZWN0X3VyaT1odHRwczovL2V4YW1wbGUuY29tJnJlc3BvbnNlX3R5cGU9dG9rZW4mc2NvcGU9ZnVsbCIsInRyYW5zYWN0aW9uX2lkIjoiSXlNVGVJSHciLCJzY29wZSI6WyJmdWxsIl19
+
+End users can manage third-party apps that they have authorized in their profile settings. The user can view authorized apps in their profile.
```

Comment:
Can you show a screenshot and instructions on how to get to the profile settings?